recipes/doxygen/all/conanfile.py

```diff
@@ -85,6 +94,13 @@ def package(self):
     def package_id(self):
         del self.info.settings.compiler
 
+        # Doxygen doesn't make code. Any package that will run is ok to use.
+        # It's ok in general to use a release version of the tool that matches the
+        # build os and architecture.
+        compatible_pkg = self.info.clone()
+        compatible_pkg.settings.build_type = 'Release'
```

Comment:
```suggestion
        compatible_pkg.settings.build_type = "Release"
```

Reply:
Suggestion: If style is going to be reviewed, an automated check would be helpful; I could have fixed this much earlier. `flake8` with [flake8-quotes](https://pypi.org/project/flake8-quotes/) would enforce this.